File: adventjs/2025/03/solution.ts
function drawGift(size: number, symbol: string): string {
  if (size < 2) return ""
  
  const lines: string[] = []
  
  // Top row
  lines.push(symbol.repeat(size))
  
  // Middle rows
  for (let i = 0; i < size - 2; i++) {
    lines.push(symbol + " ".repeat(size - 2) + symbol)
  }
  
  // Bottom row
  lines.push(symbol.repeat(size))
  
  return lines.join("\n")
}

File: adventjs/2025/03/solution.py
def draw_gift(size, symbol):
  if size < 2:
    return ""
  
  lines = []
  
  # Top row
  lines.append(symbol * size)
  
  # Middle rows
  for _ in range(size - 2):
    lines.append(symbol + " " * (size - 2) + symbol)
  
  # Bottom row
  lines.append(symbol * size)
  
  return "\n".join(lines)
